### PR TITLE
added solution part-1 dao

### PR DESCRIPTION
**Pull request by IgorBochkor**
added solution part-1 dao

### CODE REVIEW

**Review by Baldinini — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: what is missing?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (Exception e) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let's narrow this
we may encounter NumberFormatException or ArrayIndexOutOfBoundException

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                System.out.println("Будь ласка введіть коректні дані");
+            }
+            betDao.add(bet);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let’s save  the bet if it is not null

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]:
```suggestion
            System.out.println(bet);
```

File: src/main/java/core/basesyntax/dao/UsedDaoImpl.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+
+public class UsedDaoImpl implements UserDao {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/dao/UsedDaoImpl.java:7]:
```suggestion
public class UserDaoImpl implements UserDao {
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 1 above)
Comment (IgorBochkor) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: access modifier

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 2 above)
Comment (IgorBochkor) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: done!

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 3 above)
Comment (IgorBochkor) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: done!

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 4 above)
Comment (IgorBochkor) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: done

File: src/main/java/core/basesyntax/dao/UsedDaoImpl.java (hunk 5 above)
Comment (IgorBochkor) [reply src/main/java/core/basesyntax/dao/UsedDaoImpl.java:7]: sorry, what a problem?

Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/dao/UsedDaoImpl.java:7]: not UseD, but UseR

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**